docker-qgis/utils.py

```diff
@@ -752,8 +750,8 @@ def format(self, record: logging.LogRecord) -> str:
         if isinstance(record.args, dict):
             for k in record.args.keys():
                 record.args[k] = self.redact(record.args[k])
-        else:
-            record.args = tuple(self.redact(arg) for arg in record.args)
+        elif isinstance(record.args, tuple):
+            record.args = tuple(self.redact(str(arg)) for arg in record.args)
```

Comment:
```suggestion
        elif isinstance(record.args, tuple):
            record.args = tuple(self.redact(str(arg)) for arg in record.args)
        else:
            raise NotImplementedError()
```
I think this is a bit safer.